src/update/updatetypes.h

```diff
@@ -25,11 +25,15 @@
 #include <string>
 
 namespace mu::update {
+using ReleasesNotesList = std::vector<std::pair<std::string /*version*/, std::string /*notes*/> >;
```

Comment:
Maybe declare it as a struct instead of a std::pair? That way it could also be reused in ReleaseInfo